core/src/main/scala/scala/pickling/pickler/Date.scala

```diff
@@ -15,7 +15,7 @@ trait DatePicklers extends PrimitivePicklers {
     }
     private def dateFormat = dateFormatTemplate.clone.asInstanceOf[SimpleDateFormat]
 
-    def tag = FastTypeTag[Date]
+    def tag = FastTypeTag[Date]("java.util.Date")
```

Comment:
Great question!

Previously: This would use runtime reflection to generate the type key.  I've removed that posibility in favor of always using the macros.   However, we can't use the macro because it's in the same compilation unit.   I'd like to actually pull FastTypeTag into its own artifact at some point, which would clean up this usage.  For now, that's too much.